src/styles/variables.ts

```diff
@@ -140,8 +140,8 @@ export default {
     signInLogoWidthPill: 132,
     tabSelectorButtonHeight: 40,
     tabSelectorButtonPadding: 12,
-    lhnLogoWidth: 108,
-    lhnLogoHeight: 28,
+    lhnLogoWidth: 95,
+    lhnLogoHeight: 22.33,
```

Comment:
These values are from Figma
![image](https://github.com/Expensify/App/assets/85628901/78224526-58bf-4d89-82a9-e8ebddb6f877)

Reply:
Do we need to set a height or can we just set a width and the height will happen automatically?